src/api/Controllers/LayerController.cs

```diff
@@ -20,15 +20,15 @@ public LayerController(BdmsContext context, ILogger<LayerController> logger)
     /// <summary>
     /// Asynchronously gets all <see cref="Layer"/>s.
     /// </summary>
-    [HttpGet("")]
+    [HttpGet]
     public async Task<ActionResult<IEnumerable<Layer>>> GetAllAsync() => await context.Layers.AsNoTracking().ToListAsync().ConfigureAwait(false);
 
     /// <summary>
     /// Asynchronously gets the <see cref="Layer"/>s, filtered by <paramref name="profileId"/>.
     /// </summary>
     /// <param name="profileId">The id of the stratigraphy containing the layers to get.</param>
     /// GET /api/v2/layer/profileId/{profileId}
-    [HttpGet("profileId")]
+    [HttpGet("profileId/{profileId}")]
     public async Task<ActionResult<IEnumerable<Layer>>> GetByProfileIdAsync(int profileId)
```

Comment:
REQ: Ich finde diese Route ein bisschen komisch bzw. unnatürlich (e.g. `layer/profileId/1234`) und fand die Route von vorher besser. Gibt es einen Grund, den Parameter nicht als Query-Parameter zu übergeben? z.B. `layer?profileId=123`.

Reply:
Ich hatte Probleme mit muliple matches für die Route. Hast du einen Tipp?

Reply:
Wenn ich mich nicht täusche musst du den Parameter in diesem Fall mit `[FromQuery]` attributieren. Beispiel:
```csharp
[HttpGet]
public async Task<ActionResult<IEnumerable<Layer>>> GetByProfileIdAsync([FromQuery] int profileId)
```

Können es sonst kurz gemeinsam anschauen.